train_gpt2.cu

```diff
@@ -1671,6 +1672,7 @@ int main(int argc, char *argv[]) {
     }
 
     // train
+    char* mfu_str = (char*)malloc(16);
```

Comment:
I'm a bit nervous about introducing this, hardcoding 16 here, having to free it, etc.

Reply:
I agree there is a slight overhead but this is done once and left. Plus even if we had a leak it's 16 bytes :D

Reply:
Lmk if you have a better proposal! I still think in net it's better than having -100% for non-supported GPUs.

Reply:
can't we just put these 16 bytes on the stack?

Reply:
definitely - that's cleaner!